cityofaustin/formstack-alpha-theme
# Issue 4124: "Save and Resume" page

https://github.com/cityofaustin/techstack/issues/4124

**How can this be tested?**
Make sure design matches mocks. Lots of new scss.

- "Save and Resume" button
- The modal that pops up
- The page it takes you to
- Error styling if you don't enter an email
- The confirmation page when you do enter an email

A long example:
https://cityofaustin.formstack.com/forms/4124_save_and_resume_long
A short example:
https://cityofaustin.formstack.com/forms/4124_save_and_resume_styling
